components/org.wso2.carbon.editor.log.appender/src/test/resources/testng.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Please change the licence header's year

Reply:
Fixed with commit [#11d18ac2bd9d5463a9b2562066d005e5472a7ca0](https://github.com/pcnfernando/carbon-analytics/commit/11d18ac2bd9d5463a9b2562066d005e5472a7ca0)